package.json

```diff
@@ -23,6 +23,7 @@
     "typescript-demo-lib": "ts-node -r tsconfig-paths/register src/bin/typescript-demo-lib.ts"
   },
   "dependencies": {
+    "jest-mock-process": "^1.4.0",
```

Comment:
This should go into `devDependencies`. Like `npm install --save-dev`. Can you move it in to the `devDependencies` section.